Resources/Private/Partials/Backend/Handler/DoktypeDefaultHandler/Node/TypeInformation.html

```diff
@@ -9,9 +9,12 @@
 
 
 <f:comment>Get title from TCA typeColumn</f:comment>
-<f:if condition="{settings.configuration.TCA.{node.raw.table}.ctrl.typeicon_column}">
-    <f:variable name="typeColumn" value="{settings.configuration.TCA.{node.raw.table}.ctrl.typeicon_column}"/>
+<f:if condition="{settings.configuration.TCA.{node.raw.table}.ctrl.type}">
```

Comment:
This change is not correct, @see https://docs.typo3.org/m/typo3/reference-tca/10.4/en-us/Ctrl/Index.html#typeicon-column

Reply:
https://docs.typo3.org/m/typo3/reference-tca/10.4/en-us/Ctrl/Index.html#type says typeColumn is defined via `type`. For  the most common tables `tt_content` and `pages` `typeicon_column` is identical to type, however for typed records it is not necessary to define a `typeicon_column` as icons are optional. 

Thus, while this change has zero impact on tt_content and pages it is actually more correct in correspondence to the TCA reference and thus is probably more robust, especially if we add previews for other objects (like not only news items or anything).

But alas, if that is wrong I can of course revert it, but after reading it again, I still thought it was for the better

Reply:
IconFactory::mapRecordTypeToIconIdentifier only uses typeicon_column, 
The TCA from sys_file only defines typeicon_column, in TCA of sys_language there is also a mask defined. 
> "records it is not necessary to define a typeicon_column as icons are optional."
Yes, if not defined it shouldn't be shown.

BTW there is a icon already in header and the ViewHelper for this functionality is <core:iconForRecord table="{node.raw.table}" row="{node.raw.entity}" />